### ISSUE TITLE
Start of the week not changing

### ISSUE THREAD

**Issue by shravansukumar** [jonathantribouharet/JTCalendar#347]

**Comment by kishykumar** [comment]
@jonathantribouharet Do you have a solution for this? It doesn't work for me either.

**Comment by jonathantribouharet** [comment]
Yes, this attribute doesn't work (don't know why), you have to use the timezone and locale.

```
_calendarManager.dateHelper.calendar.timeZone = [NSTimeZone timeZoneWithAbbreviation:@"CDT"];
_calendarManager.dateHelper.calendar.locale = [NSLocale localeWithLocaleIdentifier:@"fr_FR"];
[_calendarManager reload];
```

**Comment by kishykumar** [comment]
@jonathantribouharet this doesn't work either. Here is my code: 

        self.calendarManager = JTCalendarManager()
        self.calendarManager.delegate = self
        self.calendarManager.menuView = calendarMenuOutlet
        self.calendarManager.contentView = calendarOutlet
        self.calendarManager.setDate(Date())
        
        var calendar = self.calendarManager.dateHelper.calendar()
        calendar!.locale = Locale(identifier: "fr_FR")
        calendar!.timeZone = TimeZone.init(abbreviation: "CDT")!
        calendar!.firstWeekday = 4 // Wednesday
        
        computeStartEndWeek(Date())
        self.calendarManager.reload()

**Comment by jonathantribouharet** [comment]
Sorry, I wasn't clear, the attribute `firstWeekday` never worked (for me).
So I don't know how to set a chosen firstWeekday manually.
In general, when you want to change the firstWeekday it's because you want your users to have the right first weekday depending of where they live so it's more simple to change the timezone by the timezone of your user which work.

**Comment by kishykumar** [comment]
@jonathantribouharet I have a different use case. I am showing the weekly earnings for the service providers for 7 days from Wednesday to Tuesday. So, I want to show Wednesday as the first week day. 
Do you know of any way to achieve this?

**Comment by jonathantribouharet** [comment]
The code managing this is [here](https://github.com/jonathantribouharet/JTCalendar/blob/master/JTCalendar/JTDateHelper.m#L113).
I didn't find how to do it before, baybe it's just an option missing, if you find it I will update the lib.

**Comment by ph1lb4** [comment]
For me it worked when I changed the `firstWeekday` property of the NSCalendar instance.

**Comment by svmrajesh** [comment]
**Swift :** 

Set your timezone and localisation while initialise calendar.

```
calendarManager = JTCalendarManager.init(locale: Locale(identifier: "en"), andTimeZone: TimeZone.current)
calendarManager!.delegate = self
calendarManager!.settings.weekDayFormat = .single;
calendarManager!.reload()
```

**Comment by ghost** [comment]
Hi, This can be fixed easily if we add one setting method in JTDateHelper.
- (void) setFirstDayOfWeek:(int) day {
    weekStartDay = day;
}
I am currently using this in my code. But i have to write this separately as it is not present in github code. Can we add it in m ain code. I have tested it is working if someone wants to have different week start day irrespective of their timezone. @jonathantribouharet
